docs/pages/docs/faq.mdx

```diff
@@ -52,6 +52,174 @@ defined and incompatible interfaces - which are the biggest problems in common
+   anything outside of Quint's fragment is only written by people with a strong
+   mathematical mindset, and those people will probably appreciate TLA+'s
+   mathematical syntax much better.
+5. Quint has the concept of a `run` to define executions to be used as tests.
+   There is no corresponding feature in TLA+.
+6. TLA+ has model values and symmetry sets, which are not available in Quint at
+   this time.
+ 
+On a less concrete note, TLA+ supporters will often say that the mathematical
+syntax helps to put people in the correct mindset of formal specification,
+while Quint as a project hopes to teach people this mindset without a big syntax
+and tool disruption.
+</FAQBox>
+
+<FAQBox title="Is Quint the same thing as PlusCal?">
+No. Although both Quint and PlusCal have the goal of making TLA+ more
+approachable to engineers, the strategies are very different. PlusCal uses the
+same tooling and similar syntax to TLA+, but introduces imperative constructs
+(while TLA+ is fully declarative). Quint, on the other hand, introduces new
+tooling and syntax, but keeps the declarative aspect.
+
+It would be possible to add the imperative constructs from PlusCal to Quint. We
+currently don't have a lot of clarity on how useful that would be.
+
+We had experiences were the expressiveness constraints imposed by PlusCal were a
+problem for some of our systems, while Quint has always been enough.
+</FAQBox>
+
+
+<FAQBox title="How does Quint compare to Alloy?">
+
+Alloy is very focused on sets, and is good for specifying requirements and data
+structures, while Quint is natively time oriented, fitting really well with
+concurrent and distributed systems. As everything in Alloy is a set/relation, it
+requires a quite abstract and specific way of modeling. 
+
+While Alloy 6 added temporal operators to handle time, they were not there for a
+long while, requiring complex workarounds to handle passage of time, and
+therefore having the language be used mostly for other things.
+
+Alloy verification can be faster than Quint for specifications without time, as
+it can use SAT solving instead of SMT solving, and the tooling looks very
+different, with Alloy Analyzer being very visual.
+
+</FAQBox>
+
+<FAQBox title="How does Quint compare to Coq/Isabelle/Lean?">
+
+Coq, Isabelle and Lean are proof assistants. They will assist you to write a
+proof, but you still have to do most of the work. Quint uses model checking,
+where you only need to define the model and the properties, and the verification
+process is fully automated. Learning and mastering how to use a proof assistant
+is much harder (usually taking years) than learning and mastering Quint.
+
+Proof assistants require you to write deductive proofs, while model checkers (as
+in Quint), use an inductive approach, exploring all reachable states in a model.
+Proof assistants are mostly used for research, and model checking techniques are
```

Comment:
don't really critical systems get proven though? 

and isn't a lot of focus in like cryptography and ethereum on using proof assistants? we may want to add something to that effect.

would be great for each to add some examples of where they have been used successfully to also ground the comparison

Reply:
From what I gathered, the meaningful industry use cases is like years of work from several experts. I'll try to find concrete information and add some "they did this but it took this big effort/expertise". In general, CompCert is the biggest example. For ethereum stuff, I need to check how many things are actually done (as opposed to ongoing research).

Reply:
one more: MatterLabs verifying the zk-verifier https://eprint.iacr.org/2024/768.pdf

Reply:
Cool, thanks! Will add those and look for more